app/code/AKisilenko/ModuleLesson6/Controller/Submit/Index.php

```diff
@@ -0,0 +1,61 @@
+<?php
+namespace AKisilenko\ModuleLesson6\Controller\Submit;
+use Magento\Framework\Controller\ResultFactory;
+use Magento\Framework\Exception\LocalizedException;
+class Index extends \Magento\Framework\App\Action\Action
+{
+    const STATUS_ERROR = 'Error';
+    const STATUS_SUCCESS = 'Success';
+    /**
+     * @var \Magento\Framework\Data\Form\FormKey\Validator
+     */
+    private $formKeyValidator;
+    /**
+     * Index constructor.
+     * @param \Magento\Framework\Data\Form\FormKey\Validator $formKeyValidator
+     * @param \Magento\Framework\App\Action\Context $context
+     */
+    public function __construct(
+        \Magento\Framework\Data\Form\FormKey\Validator $formKeyValidator,
+        \Magento\Framework\App\Action\Context $context
+    ) {
+        parent::__construct($context);
+        $this->formKeyValidator = $formKeyValidator;
+    }
+    /**
+     * @return \Magento\Framework\App\ResponseInterface|\Magento\Framework\Controller\ResultInterface
+     */
+    public function execute()
+    {
+        $request = $this->getRequest();
+        try {
+            if (!$this->formKeyValidator->validate($request) || $request->getParam('hideit')) {
+                throw new LocalizedException(__('Something went wrong. Probably you were away for quite a long time already. Please, reload the page and try again.'));
+            }
+            if (!$request->isAjax()) {
+                throw new LocalizedException(__('This request is not valid and can not be processed.'));
+            }
+            if ($request->getParam('time_cookie') === 'time_cookie') {
+                $data = [
```

Comment:
Should throw exception here like in the previous `if` statements

Reply:
Not completed... Don't see any exception statements here like `throw new LocalizedException`